spec/features/h2_object_creation_spec.rb

```diff
@@ -31,7 +31,12 @@
 
     # Adds user to depositor field
     click_button '+ Add another depositor'
+    # wait a second for the SunetID search field to display
+    sleep 1
```

Comment:
I don't understand why this is necessary. Nothing around this changed.

Reply:
I pulled this branch and commented this line out and the test still works, so can it be removed?

Reply:
Thanks for testing it and confirming it worked without the sleep line. Have removed. Also at this step, do you know why I might sometimes getting the following and other times it successfully does a lookup? 
![Screen Shot 2022-08-22 at 11 12 13 AM](https://user-images.githubusercontent.com/1619369/185957638-0bc66be1-ccfa-48a8-9423-b0db6f5f3dea.png)

Reply:
I saw that too once. This is also a problem that we've seen in production before too. I'm not sure they are connected.